services/movie_session.py

```diff
@@ -0,0 +1,65 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+import datetime as dt
+
+
+def create_movie_session(
+        movie_show_time: dt.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(
+        session_date: dt.datetime = None
+) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+    if session_date:
+        movie_session = MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+    return movie_session
+
+
+def get_movie_session_by_id(session_id: int) -> QuerySet:
```

Comment:
It returns object(MovieSession) not QuerySet

Reply:
pls, resolve comment